system/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.4.3",
   "description": "A buildpack to build node functions",
   "dependencies": {
-    "@projectriff/message": "^1.0.0"
+    "@projectriff/message": "^1.0.0",
+    "winston": "^3.2.1"
```

Comment:
Does Winston have type maps?  3rd party provided?  Roll our own?  (That's what we did for Cloudevent early on.)

Reply:
Winston seems to have types baked in: https://github.com/winstonjs/winston/blob/master/lib/winston/config/index.d.ts

Reply:
Cool.  On second thought, we probably want our own logging API (+ types) atop this one.  We don't want to expose all aspects of the underlying logging API, eg log config.

Reply:
I found this too: https://github.com/winstonjs/winston/blob/master/index.d.ts

Reply:
I was thinking the same thing, @cwallsfdc

Reply:
https://github.com/winstonjs/winston/blob/master/index.d.ts